demisto_sdk/commands/common/hook_validations/pack_unique_files.py

```diff
@@ -24,7 +24,9 @@
 from demisto_sdk.commands.common.errors import Errors
 from demisto_sdk.commands.common.hook_validations.base_validator import \
     BaseValidator
-from demisto_sdk.commands.common.tools import (get_core_pack_list, get_json,
+from demisto_sdk.commands.common.tools import (get_core_pack_list,
+                                               get_current_tags,
+                                               get_current_usecases, get_json,
```

Comment:
no need, there is already tools imported in this file

Reply:
removed functions that I added